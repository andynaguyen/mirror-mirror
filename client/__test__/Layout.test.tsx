import * as React from 'react';
import * as ReactDOM from 'react-dom';
import Layout from '../Layout';

describe('<Layout />', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Layout />, div);
  });
});
