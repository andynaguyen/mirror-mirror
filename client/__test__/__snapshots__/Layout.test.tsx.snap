// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render the contents of the layout component 1`] = `
<div>
  Dec 31st, 1969
   
  04:00:00
   
  Wednesday
   
</div>
`;
